### ISSUE TITLE
Unable to do Path Based Routes in OOv3

### ISSUE THREAD

**Issue by robtaireeo** [fabric8io/fabric8-console#12]
It looks like the base href in the index.html is causing issues for loading resources when there is a path based route.

I would like to be able to run the console and access it as so: <master/node hostname>/fabric8

Is this possible?

**Comment by gashcrumb** [comment]
We'd need the server to re-write the base href tag in index.html when it's served out, think that'd be possible @jimmidyson?

**Comment by jimmidyson** [comment]
Yeah we can do that somehow I'm sure. I can add that as a flag to kuisp & update the value of the base tag correctly.
